prosody/Dockerfile

```diff
@@ -1,10 +1,30 @@
 FROM jitsi/base
 
+ADD https://prosody.im/files/prosody-debian-packages.key /tmp/key
+
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/master/resources/prosody-plugins/token/util.lib.lua /prosody-plugins/token/util.lib.lua
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/master/resources/prosody-plugins/mod_token_verification.lua /prosody-plugins/mod_token_verification.lua
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/master/resources/prosody-plugins/mod_auth_token.lua /prosody-plugins/mod_auth_token.lua
+
 RUN \
 	apt-dpkg-wrap apt-get update && \
-	apt-dpkg-wrap apt-get install -t stretch-backports -y prosody && \
+	apt-dpkg-wrap apt-get install -y gnupg lua5.2 liblua5.2-dev lua-expat lua-socket lua-filesystem ssl-cert luarocks gcc libssl1.1 libssl1.0-dev git
+
+RUN \
+	echo deb http://packages.prosody.im/debian stretch main | tee -a /etc/apt/sources.list && \
+	apt-key add /tmp/key && \
+	apt-dpkg-wrap apt-get update && \
+	apt-dpkg-wrap apt-get install -y prosody && \
 	apt-cleanup && \
-	rm -rf /etc/prosody
+	rm -rf /etc/prosody /tmp/key
+
+RUN \
+	luarocks install luaossl && \
+	luarocks install luajwt && \
+	luarocks install luajwtjitsi && \
+	luarocks install jwt-jitsi && \
+	luarocks install luasec && \
+	luarocks install lua-cjson 2.1.0-1
```

Comment:
Let's use the debian package: https://packages.debian.org/stretch/lua-cjson

Reply:
But because we are using lua 5.2 now I have to lock the version to 2.1.0-1. Otherwise I get this error while building.

```
lua_cjson.c: In function 'json_append_data':
lua_cjson.c:743:19: warning: implicit declaration of function 'lua_objlen' [-Wimplicit-function-declaration]
             len = lua_objlen(l, -1);
```

And this one while running the container:
```
error loading module 'cjson' from file '/usr/local/lib/lua/5.2/cjson.so':
/usr/local/lib/lua/5.2/cjson.so: undefined symbol: lua_objlen
```

Reply:
Do you see this error when using the Debian packages themselves?

Reply:
Yes, I think I copied these errors from exactly this build.

Reply:
Looks like you still have some old module there. Debian won't install the package in /usr/local

Reply:
I just noticed that luajwt is also installing it as a dependency. Maybe the log was from this part of the build process.

Reply:
Hum, that's odd, why on earth doesn't it detect it's installed? Maybe because it's not installed via luarocks?